Fundamentals/Code/Umbraco-Services/index-v9.md

```diff
@@ -0,0 +1,266 @@
+{
+    public class CustomNewsArticleService: ICustomNewsArticleService
+    {
+        private readonly IMediaService _mediaService;
+        private readonly ILogger<CustomNewsArticleService> _logger;
+        private readonly IUmbracoContextFactory _contextFactory;
+
+        public CustomNewsArticleService(ILogger<CustomNewsArticleService> logger, IUmbracoContextFactory contextFactory, IMediaService mediaService)
+        {
+            _logger = logger;
+            _contextFactory = contextFactory;
+            _mediaService = mediaService;
+        }
+
+        public void DoSomethingWithNewsArticles()
+        {
+            using (var contextReference = _contextFactory.EnsureUmbracoContext())
+            {
+                IPublishedContentCache contentCache = contextReference.UmbracoContext.Content;
+                IPublishedContent newsSection = contentCache.GetAtRoot().FirstOrDefault().Children.FirstOrDefault(f => f.ContentType.Alias == "newsSection");
+                if (newsSection == null)
+                {
+                    _logger.LogDebug("News Section Not Found");
+                }
+            }
+            // etc
+        }
+    }
+}
+```
+
+## Services available
+There is full API coverage of all Umbraco core entities:
+
+- [AuditService](../../../Reference/Management/Services/AuditService/index-v9.md)
+- [ConsentService](../../../Reference/Management/Services/ConsentService/index-v9.md)
+- [ContentService](../../../Reference/Management/Services/ContentService/index-v9.md)
+- [ContentTypeService](../../../Reference/Management/Services/ContentTypeService/index-v9.md)
+- [DataTypeService](../../../Reference/Management/Services/DataTypeService/index-v9.md)
+- [EntityService](../../../Reference/Management/Services/EntityService/index-v9.md)
+- [FileService](../../../Reference/Management/Services/FileService/index-v9.md)
+- [LocalizationService](../../../Reference/Management/Services/LocalizationService/index-v9.md)
+- [MacroService](../../../Reference/Management/Services/MacroService/index-v9.md)
+- [MediaService](../../../Reference/Management/Services/MediaService/index-v9.md)
+- [MemberService](../../../Reference/Management/Services/MemberService/index-v9.md)
+- [MemberTypeService](../../../Reference/Management/Services/MemberTypeService/index-v9.md)
+- [MemberGroupService](../../../Reference/Management/Services/MemberGroupService/index-v9.md)
+
+
+### More information
+- [Umbraco Services API reference](../../../Reference/Management/Services/)
+- [Umbraco Events reference](../../../Reference/Events/) <!--- (this should change) -->
+- [Routes and controllers](../../../Reference/Routing/)
+
+### Umbraco TV
+- [Chapter: Content API](https://umbraco.tv/videos/umbraco-v7/developer/fundamentals/content-api/)
+- [Chapter: Media API](https://umbraco.tv/videos/umbraco-v7/developer/fundamentals/media-api/)
+- [Chapter: Member API](https://umbraco.tv/videos/umbraco-v7/developer/fundamentals/member-api/)
```

Comment:
```suggestion

```

Reply:
Lets just remove the videos for V7, as i don't think it makes sense that we have videos for V7 in a article for V9, plus we are going to move away from Umbraco TV anyways so the links wont work anyways